### DISCUSSION TITLE
[Core Protocol] L1Block Predeploy -- use cases?

### DISCUSSION THREAD

**Discussion by ckai0011** [ethereum-optimism/developers#453] [Core Protocol]
### Did you check the documentation?

- [X] Yes

### Did you read the specs?

- [X] Yes
- [ ] No

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

Hi -- how is the L1Block predeployed used in the OP protocol?  Can the L1Block deploy be used to manage wallet keys on L1, meaning what data is available from the L1Block contract?  Thank you.

**Comment by hash1go** [comment]
Hello!
For information available from the L1Block contract, please refer to the actual Solidity code.
https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/L2/L1Block.sol

**Comment by imtipi** [comment]
https://specs.optimism.io/protocol/predeploys.html#predeploys
